<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>867</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RequestShow</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>551</width>
     <height>671</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="RequestClose">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>810</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>121</width>
     <height>46</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>처리상태</string>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>61</width>
     <height>46</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>제목</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NewWindowRequest">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>70</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>새로고침</string>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>41</width>
     <height>46</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
